src/configs/battery.rs

```diff
@@ -1,40 +1,39 @@
-use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+use crate::config::{ModuleConfig, RootModuleConfig};
 
-use ansi_term::{Color, Style};
 use starship_module_config_derive::ModuleConfig;
 
 #[derive(Clone, ModuleConfig)]
 pub struct BatteryConfig<'a> {
-    pub full_symbol: SegmentConfig<'a>,
-    pub charging_symbol: SegmentConfig<'a>,
-    pub discharging_symbol: SegmentConfig<'a>,
-    pub unknown_symbol: Option<SegmentConfig<'a>>,
-    pub empty_symbol: Option<SegmentConfig<'a>>,
-    pub display: Vec<BatteryDisplayConfig>,
+    pub full_symbol: &'a str,
+    pub charging_symbol: &'a str,
+    pub discharging_symbol: &'a str,
+    pub unknown_symbol: &'a str,
+    pub empty_symbol: &'a str,
```

Comment:
I think it would be better written as `Option<&'a str>` as `battery` module will not be displayed when battery status is `UNKNOWN` or `EMPTY`.